test/remapping.jl

```diff
@@ -412,53 +391,104 @@ end
         end
 
         @testset "Multiple elements" begin
-            quad1 = Spaces.Quadratures.GL{1}() # FV specification
-            n1 = 2
-            mesh1 = Meshes.IntervalMesh(domain; nelems = n1)
-            topo1 = Topologies.IntervalTopology(mesh1)
-            fv_space = Spaces.SpectralElementSpace1D(topo1, quad1)
-
-            nq2 = 2
-            quad2 = Spaces.Quadratures.GLL{nq2}() # Spectral
-            n2 = 3
-            mesh2 = Meshes.IntervalMesh(domain; nelems = n2)
-            topo2 = Topologies.IntervalTopology(mesh2)
-            se_space = Spaces.SpectralElementSpace1D(topo2, quad2)
+            space1 = fv_space(domain, 2)
+            space2 = se_space(domain, 2, 3)
 
             # FV -> SE
-            R = LinearRemap(se_space, fv_space)
+            R = LinearRemap(space2, space1)
             @test nnz(R.map) == 8
             @test conservative(R)
             @test consistent(R)
             @test monotone(R)
 
             # SE -> FV
-            R = LinearRemap(fv_space, se_space)
+            R = LinearRemap(space1, space2)
             @test nnz(R.map) == 8
             @test conservative(R)
             @test consistent(R)
             @test monotone(R)
 
-            nq2 = 3
-            quad2 = Spaces.Quadratures.GLL{nq2}() # Spectral
-            n2 = 3
-            mesh2 = Meshes.IntervalMesh(domain; nelems = n2)
-            topo2 = Topologies.IntervalTopology(mesh2)
-            se_space = Spaces.SpectralElementSpace1D(topo2, quad2)
+            space2 = se_space(domain, 3, 3)
 
             # FV -> SE
-            R = LinearRemap(se_space, fv_space)
+            R = LinearRemap(space2, space1)
             @test nnz(R.map) == 12
             @test conservative(R)
             @test consistent(R)
             @test !monotone(R)
 
             # SE -> FV
-            R = LinearRemap(fv_space, se_space)
+            R = LinearRemap(space1, space2)
             @test nnz(R.map) == 12
             @test conservative(R)
             @test consistent(R)
             @test !monotone(R)
         end
     end
 end
+
+@testset "Arbitrary Order Spectral Elements Remapping" begin
```

Comment:
If you have additional ideas for how to test these operators please let me know. It may be worth calculating `R_true` by hand for one or two cases.

Reply:
I would call this testset "Spectral Elements <-> Spectral Elements Remapping", so that we have all 3 testset names consistent. 
I am going to think about this a bit more